### DISCUSSION TITLE
Failed to connect your GitHub account to Discord

### DISCUSSION THREAD

**Discussion by MaorIdi** [community/community#22197] [New to GitHub]
<p>Whenever I try to connect my GitHub account to discord it gives me an <strong>ERROR</strong> saying “Failed to connect your GitHub account to Discord”.<br>
In the GitHub account settings it shows up like my Discord account is connected to my GitHub and has access to it but It doesn’t show in my discord application, can someone please help me?</p>

**Comment by canuckjacq** [comment]
<p>:wave: Welcome!</p>
<p>This can happen if you have more than one GitHub account, using the same email address (obfuscated, like <code>myemail@domain.ooo</code> and <code>myemail+new@domain.ooo</code>) and try to connect both to a 3rd party service.</p>
<p>If that’s the case, you probably want to check and see if you can login using your other GitHub account.</p>
<p>If it’s not, you’ll want to speak to Discord.</p>

**Comment by MaorIdi** [comment]
<p>I don’t but thank you anyway, I will try and ask Discord for help :slight_smile:</p>

**Comment by kyoxxa** [comment]
<p>Did discord reply yet?</p>

**Comment by MaorIdi** [comment]
<p>Nope, nothing yet :confused:</p>

**Comment by brandao07** [comment]
<p>Having the same issue, let me know if you find a solution :grinning:</p>

**Comment by rpxs** [comment]
<p>Hi Maorldi! I had the same issue a while back and I found that when I logged out, then logged back into my Discord account, I was able to link my GitHub account correctly. I hope this helps!</p>

**Reply by macronfx** [reply]
This worked for me

**Comment by brandao07** [comment]
<p>I installed discord again and it worked for me</p>

**Comment by MaorIdi** [comment, marked as answer]
<p>Hello Everyone, I found the solution for this problem.</p>
<p>No worries, just follow these steps, and should fix the problem in no time!<br>
Go ahead and log out of both Discord and GitHub, After that clear the browser’s cache/try incognito or other browsers. Then log back into the GitHub/Discord account, then try to connect it again.</p>
<p>It should fix the problem, good luck everyone!</p>

**Comment by yahyabhatti10** [comment]
If you have an discord app on your machine and you are connecting it through that app, the chrome will open your github authorization window at your browser where your github account is login, so it will try to connect your Discord's Browser Account to Github's Browser Logged In Account and it will show you error because you have'nt logged in to your Discord Account through your browser. In easy words, you just have to login your discord account through your browser in which you have logged in your Github Account.

**Answer:**
Go to your Browser and open discord.com and login to your discord account in your browser. And then try to connect your github account from browser. This happens because you have'nt logged in to your Discord Account through your browser.

**Steps you need to take:**
Browser >> Discord.com >> Login to Discord's account in your browser, do not open app >>  Go to User Settings >>Go to Connections Tab >> Select Github.

**Reply by Rajvardhan-Singh-Rathore** [reply]
Thankyou, it worked.
